build.gradle.kts

```diff
@@ -0,0 +1,73 @@
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat
+import org.gradle.api.tasks.testing.logging.TestLogEvent
+
+plugins {
+    id("java-gradle-plugin")
+    id("com.gradle.plugin-publish") version "0.12.0"
+    groovy //TODO remove it at the end of the migration
```

Comment:
to be removed once the plugin itself is re-written in kotlin, right?

Reply:
Yes, for now, only the build script is rewriten in kotlin. I'll remove this comment